pygerduty/__init__.py

```diff
@@ -414,6 +414,9 @@ def resolve(self, requester_id):
     def acknowledge(self, requester_id):
         self._do_action('acknowledge', requester_id=requester_id)
 
+    def snooze(self, duration):
+        self._do_action('snooze', duration=duration)
```

Comment:
Yeah, it's required when doing Token Auth which is currently the only auth mechanism supported by Pagerduty since user auth is deprecated. :)